main.js

```diff
@@ -93,10 +93,29 @@
             refreshEvents(CTLEventsManager.filteredEvents, 1);
         });
 
+        $('input[name="start_date"]').on('change', function(e) {
+            var date = e.target.value;
+            CTLEventsManager.filteredEvents =
+                CTLEventUtils.filterEventsByDateRange(
+                    CTLEventsManager.allEvents,
+                    date ? new Date(date + ' 00:00:00 GMT-0500') : null,
+                    null);
+            refreshEvents(CTLEventsManager.filteredEvents, 1);
+        });
+        $('input[name="end_date"]').on('change', function(e) {
+            var date = e.target.value;
+            CTLEventsManager.filteredEvents =
+                CTLEventUtils.filterEventsByDateRange(
+                    CTLEventsManager.allEvents,
+                    null,
```

Comment:
This null value actually needs to be the value of the start_date input - otherwise both dates aren't filtered for. Similar case on line 102.